CrazyCanvas/Source/World/LevelObjectCreator.cpp

```diff
@@ -326,9 +326,9 @@ ELevelObjectType LevelObjectCreator::CreateFlagDeliveryPoint(const LambdaEngine:
 
 	pECS->AddComponent<FlagDeliveryPointComponent>(entity, {});
 
-	uint32 teamIndex = 0;
+	uint8 teamIndex = 0;
 	size_t teamIndexPos = levelObject.Name.find("TEAM");
-	if (teamIndexPos != String::npos) teamIndex = std::stoul(levelObject.Name.substr(teamIndexPos + 4));
+	if (teamIndexPos != String::npos) teamIndex = (uint8)std::stoi(levelObject.Name.substr(teamIndexPos + 4));
```

Comment:
What's the reason to instead of using std::stoul (as used previously) to instead use std::stoi with a cast? Without knowing the reason it seems like a more roundabout way to do it.

Reply:
The primitive type of teamIndex is uint8 and it is, therefore, more "efficient" to cast an int32 to uint8, compared to unsigned long to uint8. In some architectures, the unsigned long is the same as uint32 and in that case, it makes no difference.